cargo-spatial/src/opt.rs

```diff
@@ -0,0 +1,78 @@
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+/// Build, run, and deploy SpatialOS workers written in Rust using Cargo.
```

Comment:
What happens if we have workers in other build systems? Do we want to support the whole ecosystem as well?

Reply:
I'd put that firmly in "to be determine" territory. I really don't have a good sense at this point of what it would look like to pull in third-party workers written in Rust, let alone ones written in other languages. I think for now it makes sense to limit cargo-spatial to just handling Rust workers, and we can worry about handling more complex setups later.

Reply:
I would say that `cargo spatial` should have no opinion about packaging non-Rust workers. To me, `cargo spatial build` should just map a Rust project (with all code and dependencies) into an assembly zip.